_config.yml

```diff
@@ -11,7 +11,12 @@ markdown: kramdown
 
 # Custom Variables
 
+main:
+    plugins: "assets/plugins"
+    css: "assets/css"
+    js: "assets/js"
+
 path:
```

Comment:
When I got the pages connected, the path would go to about/assets/plugins or anything to try to find those files. I wasn't sure how to get to them. I did not end up using those variables in the config.yml. It currently just goes /assets/plugins in the head.html to find everything.